setup.py

```diff
@@ -87,9 +90,10 @@
         "clickhouse-sqlalchemy>=0.1.4,<0.2",
         "sqlalchemy>=1.3,<1.4",
     ],
-    "dask": ["dask[array,dataframe]>=2021.2.0,<2022.0.0", "pyarrow>=1,<7"],
+    "dask": ["dask[dataframe,array]>=2021.2.0,<2022.0.0", "pyarrow>=1,<7"],
+    "datafusion": ["datafusion>=0.4"],
     "geospatial": [
-        "geoalchemy2>=0.6,<0.11",
+        "geoalchemy2>=0.6,<0.10",
```

Comment:
This doesn't look right. We increased the upper bound recently to allow `0.10`.

Reply:
Updated.